src/services/s3/format.ts

```diff
@@ -191,7 +191,8 @@ export default ({
     bucketOwnerName,
     bucketPolicies: bucketPolicies.map(bp => ({
       id: cuid(),
-      policy: formatIamJsonPolicy(bp.policy)
+      policy: formatIamJsonPolicy(bp.policy),
+      rawPolicy: bp.policy,
```

Comment:
I think this should be top level. Are there cases where there is more than one bucket policy? I only see one in the format function and @ckoning I dont believe is familiar with a case where there is more than one.

Reply:
I made the requested changes here. Although, these changes will change [some rules](https://github.com/cloudgraphdev/cloudgraph-policy-packs/pull/2).